### ISSUE TITLE
Add `lower_` property to `Span` class?

### ISSUE THREAD

**Issue by bdewilde** [explosion/spaCy#669]
This is a feature request, not a bug. (Is there a better place to post re: enhancements?)

For convenience, it would be great to have a `lower_` property on spans analogous to the [`lemma_`](https://github.com/explosion/spaCy/blob/master/spacy/tokens/span.pyx#L364) property. Looks like it would be a super straightforward change, maybe even just

```
cdef class Span:
    ...
    property lower_:
        def __get__(self):
            return ' '.join([t.lower_ for t in self]).strip()
```

Thanks in advance!

**Comment by honnibal** [comment]
Maybe! If so, we should support the other string methods too, right?

I think we should think more deeply about an ABC over `Token`, `Span` and `Doc`.

**Comment by bdewilde** [comment]
Agreed! I just didn't want to ask for more than I had immediate use for. :)

**Comment by bhargavvader** [comment]
I'd like to take a shot at this; any pointers?

**Comment by honnibal** [comment]
Sorry for letting this slip --- adding the other string methods sounds good. Adding the help wanted tag here.

The file to change is https://github.com/explosion/spaCy/blob/master/spacy/tokens/span.pyx

Tests would go here: https://github.com/explosion/spaCy/blob/master/spacy/tests/spans/test_span.py

**Comment by ines** [comment]
Fixed in #883!
